benchmarks/time/src/main/scala/strawman/collection/immutable/ImmutableArrayBenchmark.scala

```diff
@@ -10,7 +10,7 @@ import scala.Predef.intWrapper
 
 @BenchmarkMode(scala.Array(Mode.AverageTime))
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
-@Fork(1)
+@Fork(2)
```

Comment:
Why are you changing this?

Reply:
@Ichoran I just though fork 2 jvm would make result more stable. (with less error). Reverted.